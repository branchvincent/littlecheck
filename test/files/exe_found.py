#!/bin/bash
# REQUIRES: command -v bash
echo fart
ath
