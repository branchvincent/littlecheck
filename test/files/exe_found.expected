Failure in exe_found.py:

  There were no remaining checks left to match stdout:1:
    fart

  additional output on stderr:1:
    exe_found.py: line 4: ath: command not found

  Context:
    fart <= no more checks

  when running command:
    /bin/bash exe_found.py
